examples/protractor/testGithubPage.js

```diff
@@ -30,6 +30,23 @@ const attachments = [
 ];
 
 describe('github homepage', function() {
+    let originalTimeout;
+
+    beforeAll(function() {
+        const func = () => {};
+
+        func();
+    });
+
+    beforeEach(function() {
+        originalTimeout = jasmine.DEFAULT_TIMEOUT_INTERVAL;
```

Comment:
Which purpose of it?

Reply:
I have the Promise inside of the test and it causes some problems, this string solves the problems